### PR TITLE
Optimize some code

### PR DESCRIPTION
**Pull request by kevinvenclovas**
1. Send feeds to multiple marketplaces
2. Add better exception handling to handle exception more efficient

### CODE REVIEW

File: Source/FikaAmazonAPI/Services/FeedService.cs [hunk 1]
```diff
@@ -127,6 +127,17 @@ public CreateFeedDocumentResult CreateFeedDocument(ContentType contentType)
...
+            return SubmitFeed(xml, feedType, new List<string> { MarketPlace.ID }, feedOptions);
+        }
+
+        /// <summary>
+        /// read full step  https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/feeds-api-use-case-guide/feeds-api-use-case-guide_2021-06-30.md
+        /// </summary>
+        /// <param name="xml"></param>
+        /// <param name="feedType"></param>
+        /// <returns></returns>
+        public string SubmitFeed(string xml, FeedType feedType, List<string> marketPlaceIds, FeedOptions feedOptions = null)
```
Comment (abuzuhri) [inline Source/FikaAmazonAPI/Services/FeedService.cs:140]: Can you change this as optional , and the default system will add current marketplace id if parameter null as you did in last pull request

Comment (abuzuhri) [reply Source/FikaAmazonAPI/Services/FeedService.cs:140]: List<string> marketPlaceIds

File: Source/FikaAmazonAPI/Services/FeedService.cs [hunk 2]
```diff
@@ -139,7 +150,7 @@ public string SubmitFeed(string xml, FeedType feedType, FeedOptions feedOptions
             {
                 FeedType = feedType.ToString(),
                 InputFeedDocumentId = feedCreate.FeedDocumentId,
-                MarketplaceIds = new List<string> { MarketPlace.ID },
+                MarketplaceIds = marketPlaceIds,
```
Comment (abuzuhri) [inline Source/FikaAmazonAPI/Services/FeedService.cs:153]: MarketplaceIds = marketPlaceIds ??new List<string> { MarketPlace.ID },

**Review by abuzuhri — CHANGES_REQUESTED** [review]
plz try to fix that

**Comment by kevinvenclovas** [comment]
fixed

**Review by abuzuhri — APPROVED** [review]

**Comment by abuzuhri** [comment]
Thanks for fix

**Pull request merged**